app/src/views/private/components/export-sidebar-detail.vue

```diff
@@ -461,7 +461,7 @@ function useUpload() {
 			});
 		} catch (err: any) {
 			notify({
-				title: t('import_data_error'),
+				title: err?.response?.data?.errors[0]?.message || err?.response?.data?.message || t('import_data_error'),
```

Comment:
The error message coming from the API isn't translated, so this won't work as nicely. Could we use the translate on the `code` instead of `message` using ``t(`errors.${code}`)`` and fallback to an `unexpectedError` call when the code is `INTERNAL_SERVER_ERROR`?

Reply:
Hey! 
Thanks for getting back on this. So the reason we wanted to introduce this change was so that we understand why our file uploads are getting rejected. This typically happens because of some database or Directus level validation failing on a particular row. In the current implementation, we see a standard "File does not match directory structure" error which isn't helpful in debugging what's wrong with the file. We also noted that the real reason for the error is actually visible when you inspect the query in the Network tab of the Developer Tools. We thus thought it would be more useful just to show that error here as it is. The drawback obviously being that this error message is not translated currently.
Are you suggesting that we read the error message from the response and attempt to translate that first? And if not, fall back to whatever we received from the backend.?

Reply:
> Are you suggesting that we read the error message from the response and attempt to translate that first? And if not, fall back to whatever we received from the backend.?

Yeah exactly 🙂 If the error is one of the known codes in the translations we can show the nicely translated one, otherwise we can fallback to showing the unexpected error dialog which will render the `v-error` component which is made to handle parsing and showing the raw error (including the message and a button to copy the error stack)